src/com/serotonin/mango/web/mvc/controller/DataPointDetailsController.java

```diff
@@ -50,7 +50,7 @@ protected ModelAndView handleRequestInternal(HttpServletRequest request,
 			HttpServletResponse response) throws Exception {
 		Map<String, Object> model = new HashMap<String, Object>();
 		User user = Common.getUser(request);
-
+		Permissions.ensureAdmin(user);

```

Comment:
Why we are blocking the "data point details" page?  
If user has at least "read" access to specific data point I think that he should have right to open this page.

Reply:
The read permission is applied to visible on view, and you can go to history, loupe on watchlist.